routing/localchans/manager.go

```diff
@@ -106,6 +127,33 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 		return err
 	}
 
+	// Return an error if any of the channels in chansToUpdate
+	// were invalid.
+	for chanPoint := range chansToUpdate {
+		channel, err := r.FetchChannel(chanPoint)
+		if errors.Is(err, channeldb.ErrChannelNotFound) {
+			errInvalidChans += fmt.Sprintf(
```

Comment:
nit: for all these `fmt.Sprint(...` statements, see contribution guidelines [here](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#additional-style-constraints-on-top-of-gofmt) basically, if all the arguments cant fit on 1 line, then the open and close brackets should also be on their own lines. so:
```
fmt.Sprintf(
<args here>,
)
```

Reply:
Fixing it.

Reply:
still need new line after the last argument before the `)`